internal/static/testdata/apps/single-app/app.go

```diff
@@ -0,0 +1,17 @@
+package app
+
+import "github.com/dogmatiq/dogma"
+
+const (
+	// AppKey is the application key.
+	AppKey = "8a6baab1-ee64-402e-a081-e43f4bebc243"
```

Comment:
I would ditch these constants everywhere other than the `const-value-ident` test, otherwise it looks like it's important to this test.

Reply:
Yep, makes sense. This is addressed in https://github.com/dogmatiq/configkit/pull/118/commits/8f8e0bb8a6d0b599290ee99f422d6e9dd433ccef.